src/test/java/org/fundacionjala/sfdc/stepdefinition/CampaignSteps.java

```diff
@@ -0,0 +1,37 @@
+package org.fundacionjala.sfdc.stepdefinition;
+
+import cucumber.api.java.en.And;
+import org.fundacionjala.sfdc.pageobjects.campaigns.CampaignInputs;
+import org.fundacionjala.sfdc.pageobjects.campaigns.SFCampNewModifyPage;
+import org.fundacionjala.sfdc.pageobjects.opportunities.OpportInputs;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.sfdc.pageobjects.opportunities.OpportInputs'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)